linux/build.gradle

```diff
@@ -54,4 +54,18 @@ def getInputDir() {
 
 def getLocalBuildStatus() {
 	return hasProperty("INPUT_DIR") ? "true" : "false"
+}
+
+def getContainerRegistry() {
+	return hasProperty("CONTAINER_REGISTRY") ? CONTAINER_REGISTRY.toString() : ""
+}
+
+def setBuildArgs(args) {
```

Comment:
Are we adding more args than just the one CONTAINER_REGISTRY build arg?

Reply:
This is the first build arg that we are adding but I created this for the future in case we want to add more build arguments.

Reply:
Recommend simplifying for now [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it)